docs/release_process_quarterly.md

```diff
@@ -0,0 +1,501 @@
+### Prerequisites
+    - [ ] jpo-ode release completed
+    - [ ] jpo-geojsonconverter release completed
+    - [ ] jpo-conflictmonitor release completed
+
+### 1. Code Ready & Release Notes
+    - [ ] Release notes drafted & added to `Release_notes.md` file in `docs` directory
+    - [ ] Code changes for release are merged into `develop`
+    - [ ] A new branch `release/(year)-(quarter)` is created from `develop`
+
+### 2. Preliminary Testing
+    - [ ] deduplicator code compiles
+    - [ ] deduplicator unit tests pass
+    - [ ] program starts up correctly
+    - [ ] program processes BSM, SPaT, Map, and TIM messages
+    - [ ] ProcessedMap, ProcessedSpat, OdeMap, OdeTim, OdeRawEncodedTim, OdeBsm messages are consumed
+    - [ ] A single message is output for each of the input types
+
+### 3. Project Reference Updates & Release Creation
+    - [ ] Update version number in pom.xml file for the `jpo-deduplicator` project if not already done.
+    - [ ] Update the corresponding version number for the jpo-geojsonconverter and jpo-ode-* dependencies in the pom.xml files of the jpo-deduplicator project. This change will be necessary in the jpo-deduplicator/pom.xml file.
+    - [ ] Update git submodule references for the ‘jpo-ode’ project to point to tagged commits in projects with updated `master` branches.
+        - [ ] Open the jpo-ode project in an IDE.
+        - [ ] Navigate to the jpo-utils directory and run `git checkout tags/jpo-utils-x.x.x` to update the submodule reference.
+    - [ ] Merge `release/(year)-(quarter)` branch into `master` branch for the jpo-deduplicator project, update the existing release tag (e.g. jpo-deduplicator-x.x.x) to point to the newly committed version of jpo-deduplicator
+    - [ ] Create docker image
+        - [ ] Use the release for the jpo-deduplicator to produce docker image with the same tag name, containing the version number.
+        - [ ] Upload docker image to [DockerHub](https://hub.docker.com/u/usdotjpoode)
+        - [ ] Tag docker image with the version number of the app. (e.g. 1.0.0)
+        - [ ] Tag docker image with year and quarter of release. (e.g. 2024-Q2)
+        - [ ] Tag docker image with 'latest' tag for the most recent release.
+    - [ ] Merge master branch into develop branch & verify that CI/CD passes.
+
+### 4. DockerHub Image Testing
+    - [ ] image starts up correctly
+    - [ ] ProcessedMap, ProcessedSpat, OdeMap, OdeTim, OdeRawEncodedTim, OdeBsm messages are consumed
+    - [ ] A single message is output for each of the input types
+
+
+## jpo-cvmanager
+### Prerequisites
+    - [ ] jpo-conflictmonitor release completed
+
+### 1. Code Ready & Release Notes
+    - [ ] Release notes drafted & added to `Release_notes.md` file in `docs` directory
+    - [ ] Code changes for release are merged into `develop`
+    - [ ] A new branch `release/(year)-(quarter)` is created from `develop`
+
+### 2. Preliminary Testing
+    - [ ] code compiles
+    - [ ] unit tests pass
```

Comment:
Unit tests for python api, webapp, and intersection Java api

Reply:
Addressed!